src/packages/@ncigdc/components/Modals/UserProfileModal.js

```diff
@@ -0,0 +1,135 @@
+  withTheme,
+  withRouter,
+);
+
+const paginationPrefix = 'userProjectsTable';
+
+const UserProfileModal = ({
+  user: { username, projects },
+  theme,
+  push,
+  query,
+}) => {
+  const flattenedProjects = Object.keys(projects).reduce(
+    (acc, k) => ({ ...acc, ...projects[k] }),
+    {},
+  );
+  const allValues = [
+    ...new Set(
+      Object.keys(flattenedProjects).reduce(
+        (acc, projectId) => [...acc, ...flattenedProjects[projectId]],
+        [],
+      ),
+    ),
+  ];
+  const searchTerm = query[`${paginationPrefix}_search`] || '';
+
+  const data = Object.keys(flattenedProjects)
+    .filter(projectId =>
+      projectId.toUpperCase().includes(searchTerm.toUpperCase()),
+    )
+    .map((projectId, i) => ({
+      project_id: <Highlight search={searchTerm}>{projectId}</Highlight>,
+      ...allValues.reduce(
+        (acc, v) => ({
+          ...acc,
+          [v]: flattenedProjects[projectId].includes(v)
+            ? <span><Check /><Hidden>True</Hidden></span>
+            : <Hidden>False</Hidden>,
+        }),
+        {},
+      ),
+    }));
+  return (
+    <BaseModal title={`Username: ${username}`} closeText="Done">
+      <Column>
+        <Row style={{ justifyContent: 'center' }}>
+          <label htmlFor="filter-cases">
+            <div
+              style={{
+                borderTop: `1px solid ${theme.greyScale5}`,
+                borderLeft: `1px solid ${theme.greyScale5}`,
+                borderBottom: `1px solid ${theme.greyScale5}`,
+                borderRight: 0,
+                borderRadius: '4px 0 0 4px',
+                backgroundColor: `${theme.greyScale4}`,
+                width: '38px',
+                height: '100%',
+                display: 'flex',
+                justifyContent: 'center',
+                alignItems: 'center',
+              }}
+            >
+              <SearchIcon size={14} />
+            </div>
+            <Hidden>filter cases</Hidden>
+          </label>
+          <input
+            id="filter-projects"
+            name="filter-projects"
+            placeholder="Search for project id"
+            type="text"
+            onChange={e => {
+              push({
+                query: {
+                  ...omit(query, `${paginationPrefix}_offset`),
+                  [`${paginationPrefix}_search`]: e.target.value,
+                },
+              });
+            }}
+            style={{
+              fontSize: '14px',
+              paddingLeft: '1rem',
+              width: '28rem',
+              borderRadius: '0 4px 4px 0',
+              border: `1px solid ${theme.greyScale5}`,
+            }}
+          />
+        </Row>
+        <Row style={{ justifyContent: 'center' }}>
+          <LocalPaginationTable
+            className="test-consequences-table"
+            data={data}
+            prefix={paginationPrefix}
+            entityName="projects"
+            style={{ width: '70%' }}
+            enablePickSize={false}
```

Comment:
Do we need to disable picking size? 10 is a pretty small number, might be better to just have overflow?

Reply:
both scrolling and pagination in a modal seems too much and also didn't want to deal with small issues like the picker being under the modal
<img width="725" alt="screen shot 2017-08-30 at 3 17 43 pm" src="https://user-images.githubusercontent.com/1314446/29890721-9430a34e-8d96-11e7-974a-d262baa55a1d.png">
maybe I'll do away with the front end pagination and see what phuong-my thinks of just scroll

Reply:
the full page you had before seems better

Reply:
reason for changing to modal was they didn't want it to be a full data page, and they wanted to "have a screen where users can see it without navigating away from current screen"